src/components/EssentialLink.vue

```diff
@@ -1,17 +1,13 @@
 <template>
   <q-item
     clickable
-    tag="a"
-    target="_blank"
-    :href="props.link"
+    :to="{ name: `${props.name}` }"
```

Comment:
não precisa do ${}, e poderia passar direto a string
```js
:to="props.name"
```

Reply:
Não seria assim elias, seria `:to="{ name: props.name }"`